src/main/main.ts

```diff
@@ -302,10 +301,10 @@ ipcMain.handle('run-follower', async () => {
   await followerDaemon();
 });
 
-ipcMain.handle('ipfs-peers', () => ipfs('swarm peers'));
+ipcMain.handle('ipfs-peers', async () => await rpcRequest('swarm/peers'));
```

Comment:
can skip async/await, function still returns promise

```suggestion
ipcMain.handle('ipfs-peers', () => rpcRequest('swarm/peers'));
```

Reply:
removed async await

Reply:
Thanks